test/serializer_bin/serializer_bin_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Membrane.WAV.SerializerBinTest do
+  use ExUnit.Case
+  use Membrane.Pipeline
+
+  import Membrane.Testing.Assertions
+
+  alias Membrane.Testing.Pipeline
+
+  @module Membrane.WAV.SerializerBin
+
+  @input_path Path.expand("../fixtures/input.wav", __DIR__)
+  @reference_path Path.expand("../fixtures/reference_processed.wav", __DIR__)
+
+  setup_all do
+    output_path = Path.join([System.tmp_dir!(), "output.wav"])
+    on_exit(fn -> File.rm!(output_path) end)
+
+    [output_path: output_path]
+  end
```

Comment:
It's a fairly recent discovery, but systematically introduced to tests among elements - you can use the `tmp_dir` tag of ExUnit https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-tmp-dir

Reply:
Thanks, didn't know this one. Shame that the created `"tmp"` directory in project dir isn't automatically removed though.